app/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,22 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```

Comment:
does the DayNight mean it will follow system preferences for dark mode? if so that's pretty sick